databricks/README.md

```diff
@@ -14,91 +14,102 @@ Monitor Databricks Spark applications with the [Datadog Spark integration][3]. I
-    - Set `<init-script-folder>` path to where you want your init scripts to be saved in.
 <!-- partial
 {{% site-region region="us,us3,us5,eu,gov,ap1" %}}
-
-3. Configure a new Databricks cluster with the cluster-scoped init script path using the UI, Databricks CLI, or invoking the Clusters API.
-    - Set the `DD_API_KEY` environment variable in the cluster's Advanced Options with your Datadog API key.
-    - Add `DD_ENV` environment variable under Advanced Options to add a global environment tag to better identify your clusters.
-    - Set `DD_SITE` to your site: {{< region-param key="dd_site" code="true" >}}.
+You can also define or modify environment variables with the cluster-scoped init script path using the UI, Databricks CLI, or invoking the Clusters API:
+  - Set `DD_API_KEY` to better identify your clusters.
+  - Set `DD_ENV` to better identify your clusters.
+  - Set `DD_SITE` to your site: {{< region-param key="dd_site" code="true" >}}. Defaults to `datadoghq.com`
 {{% /site-region %}}
 partial -->
 
+<div class="alert alert-warning">For security reasons, it's not recommended to define the `DD_API_KEY` environment variable in plain text directly in the UI. Instead, use <a href="https://docs.databricks.com/en/security/secrets/index.html">Databricks secrets</a>.</div>
+
+
+
+#### With a global init script
+
+A global init script runs on every cluster created in your workspace. Global init scripts are useful when you want to enforce organization-wide library configurations or security screens. 
+
+<div class="alert alert-info">Only workspace admins can manage global init scripts.</div>
+
+Use the Databricks UI to edit the global init scripts:
+
+1. Choose one of the following scripts to install the Agent on the driver or on the driver and worker nodes of the cluster.
+2. Modify the script to suit your needs. For example, you can add tags or define a specific configuration for the integration.
+3. Go to the Admin Settings and click the **Global Init Scripts** tab.
+4. Click on the **+ Add** button.
+5. Name the script, for example `Datadog init script` and then paste it in the **Script** field.
+6. Click on the **Enabled** toggle to enable it.
+7. Click on the **Add** button.
+
+After these steps, any new cluster uses the script automatically. More information on global init scripts can be found in the [Databricks official documentation][16].
 
-#### Standard cluster
+<div class="alert alert-info">You can define several init scripts and specify their order in the UI.</div>
 
 <!-- xxx tabs xxx -->
 <!-- xxx tab "Driver only" xxx -->
-##### Install the Datadog Agent on Driver
+##### Install the Datadog Agent on driver
+
 Install the Datadog Agent on the driver node of the cluster. 
 
-After creating the `datadog-install-driver-only.sh` script, add the init script path in the [cluster configuration page][6].
+<div class="alert alert-warning">You need to define the value of the `DD_API_KEY` variable inside the script.</div>
```

Comment:
nit:
After the whole coming soon issue we had with weaviate, torchserve and dcgm, I think that if we expect customer to look at these instructions in the tile, then we shouldn't use this div.

Reply:
Personal opinion: I do not really expect customers to look at the tile directly but rather use the documentation site because this is a tile only integration